### PR TITLE
Fs 722 v2 comments functionality

### PR DESCRIPTION
**Pull request by AbuNavsa**
- Database models and schemas created for the comments table
- Endpoints created for GET and POST of comments. GET retrieves all comments based on its column values requested. The 2 compulsory requested fields are application_id and sub_criteria_id
- Get and Post queries written and methods created
- user account data is not recieved here, will be recieved in ASSESSMENT to keep the assessment store solely assessment related

### CODE REVIEW

File: openapi/components.yml [hunk 1]
```diff
@@ -84,20 +84,36 @@ components:
...
+        application_id:
+          type: string
+        # comment:
+        #   type: string
+        # user_id:
+        #   type: string
+        # sub_criteria_id:
+        #   type: string
+        # comment_type:
+        #   type: string
```
Comment (gio-karunakaran) [inline openapi/components.yml:116]: can we remove these?

File: db/queries/comments/queries.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+from typing import List
+from db import db
+from db.models.comment.comments import Comment
+from db.schemas import CommentMetadata
+from sqlalchemy import func
+from sqlalchemy import select
+from sqlalchemy.orm import defer
+from sqlalchemy.orm import load_only
+
+# May need rewrite after testing
```
Comment (tferns) [inline db/queries/comments/queries.py:16]: small comment here left in too

File: db/models/comment/enums.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class CommentType(Enum):
+    """Status The ENUM used by `db.models.Comments` to validate the
+    possible values for the `comment_type` column."""
+
+    COMMENT = 0
```
[the remark this answers is not in the document]
Comment (AbuNavsa) [reply db/models/comment/enums.py:8]: Yep, not been given any other types yet, only examples of some for example "REPLY" would be another but not sure that has been discussed yet

File: openapi/components.yml (hunk 1 above)
Comment (AbuNavsa) [reply openapi/components.yml:116]: yep, removed

File: api/routes/comment_routes.py [hunk 4]
```diff
@@ -0,0 +1,51 @@
+from typing import Dict
+from db.queries.comments import get_comments_for_application_sub_crit
+from db.queries.comments import create_comment_for_application_sub_crit
+from db.schemas import CommentMetadata
+from flask import request
+
+def comments_for_application_sub_criteria(
+    application_id: str, sub_criteria_id: str
+) -> Dict:
+    print("😎😎😎😎😎😎😎😎")
```
[the remark this answers is not in the document]
Comment (AbuNavsa) [reply api/routes/comment_routes.py:10]: Woops!

File: db/queries/comments/queries.py (hunk 2 above)
Comment (AbuNavsa) [reply db/queries/comments/queries.py:16]: Not commented by me so unsure whether to remove

**Review by gio-karunakaran — APPROVED** [review]
LGTM 👍🏾

**Pull request merged**